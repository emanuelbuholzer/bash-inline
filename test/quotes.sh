#!/usr/bin/env bash
NAME="John"
echo "Hi $NAME"  #=> Hi John
echo 'Hi $NAME'  #=> Hi $NAME
