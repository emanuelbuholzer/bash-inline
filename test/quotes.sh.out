#!/usr/bin/env bash
NAME="John";
echo "Hi $NAME";
echo 'Hi $NAME';